packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/CustomVisConfigTabs.tsx

```diff
@@ -73,69 +416,217 @@ const CustomVisConfigTabs: React.FC = memo(() => {
+        let container = document.getElementById(containerId);
+        if (!container) {
+            const wrapper = document.createElement('div');
+            wrapper.className = 'monaco-editor';
+            container = document.createElement('div');
+            container.id = containerId;
+            wrapper.appendChild(container);
+            document.getElementById('root')?.appendChild(wrapper);
+        }
+        setMonacoOptions({
+            ...MONACO_DEFAULT_OPTIONS,
+            overflowWidgetsDomNode: container,
+        });
+    }, [monacoOptions]);
+    const { itemsMap } = useVisualizationContext();
+
+    if (!isCustomConfig) return null;
+
+    if (!monacoOptions || isLoading) {
         return <Loader color="gray" size="xs" />;
     }
 
-    if (!isCustomConfig) return null;
-    const { visSpec, setVisSpec, series } = visualizationConfig.chartConfig;
+    const { series } = visualizationConfig.chartConfig;
+
+    const isEditorEmpty = (editorConfig || '')?.length === 0;
 
     return (
-        <Tabs
-            defaultValue="config"
-            style={{ flexGrow: 1 }}
-            styles={{
-                root: {
-                    display: 'flex',
-                    flexDirection: 'column',
-                },
-                panel: {
-                    flexGrow: 1,
-                },
-            }}
-        >
-            <Tabs.List>
-                <Tabs.Tab value="config">Config</Tabs.Tab>
-                <Tabs.Tab value="data">Data</Tabs.Tab>
-            </Tabs.List>
-
-            <Tabs.Panel value="config">
-                <Editor
-                    loading={<Loader color="gray" size="xs" />}
-                    beforeMount={(monaco) =>
-                        loadMonaco(monaco, schemas.current!)
-                    }
-                    defaultLanguage="json"
-                    options={{ ...MONACO_DEFAULT_OPTIONS }}
-                    value={visSpec}
-                    onChange={(config) => setVisSpec(config ?? '')}
+        <>
+            <Flex justify="space-between" align="flex-end">
+                <SelectTemplate
+                    itemsMap={itemsMap}
+                    isCustomConfig={isCustomConfig}
+                    isEditorEmpty={isEditorEmpty}
+                    setEditorConfig={setEditorConfig}
                 />
-            </Tabs.Panel>
+                {isAiEnabled && (
+                    <>
+                        <Text>or</Text>
+                        <GenerateVizWithAi
+                            itemsMap={itemsMap}
+                            sampleResults={series.slice(0, 3)}
+                            setEditorConfig={setEditorConfig}
+                            editorConfig={editorConfig}
+                        />
+                    </>
+                )}
+                <DocumentationHelpButton href="https://docs.lightdash.com/references/custom-charts#custom-charts" />
+            </Flex>
+            {/* <Tabs
```

Comment:
previously there were tabs, one with config,a nother with data. Data is not very useful now we have autocomplete , and results are already in the table.